pennylane/beta/queuing/measure.py

```diff
@@ -19,14 +19,20 @@
 """
 import collections
 
-import pennylane as qml
 from pennylane.operation import Expectation, Observable, Probability, Sample, Variance
 from pennylane.ops import Identity
 from pennylane.qnodes import QuantumFunctionError
 
-MeasurementProcess = collections.namedtuple("MeasurementProcess", ["return_type"])
-"""NamedTuple: A namedtuple that contains the return_type of the circuit and
-whose instance can be queried by id."""
+from .queuing import QueuingContext
+
+class MeasurementProcess:
+    """A class that contains the return_type of the circuit and whose instance
+    can be queried by id."""
+
+    def __init__(self, return_type, queue_idx=None):
+        self.return_type = return_type
+        self.queue_idx = queue_idx
+        self.wires = []
```

Comment:
To be compatible with the current `CircuitGraph` object, this was made a class. Might have to reconsider when integrating with #769